DeepSpeech.ipynb

```diff
@@ -148,17 +148,17 @@
    },
    "outputs": [],
    "source": [
-    "learning_rate = float(os.environ.get('ds_learning_rate', 0.001)) # TODO: Determine a reasonable value for this\n",
+    "learning_rate = float(os.environ.get('ds_learning_rate', 0.001))\n",
     "beta1 = float(os.environ.get('ds_beta1', 0.9))                   # TODO: Determine a reasonable value for this\n",
     "beta2 = float(os.environ.get('ds_beta2', 0.999))                 # TODO: Determine a reasonable value for this\n",
     "epsilon = float(os.environ.get('ds_epsilon', 1e-8))              # TODO: Determine a reasonable value for this\n",
-    "training_iters = int(os.environ.get('ds_training_iters', 15))    # TODO: Determine a reasonable value for this\n",
-    "train_batch_size = int(os.environ.get('ds_train_batch_size', 48))# TODO: Determine a reasonable value for this\n",
-    "dev_batch_size = int(os.environ.get('ds_dev_batch_size', 32))    # TODO: Determine a reasonable value for this\n",
-    "test_batch_size = int(os.environ.get('ds_test_batch_size', 32))  # TODO: Determine a reasonable value for this\n",
-    "display_step = int(os.environ.get('ds_display_step', 1))         # TODO: Determine a reasonable value for this\n",
-    "validation_step = int(os.environ.get('ds_validation_step', 1))   # TODO: Determine a reasonable value for this\n",
-    "checkpoint_step = int(os.environ.get('ds_checkpoint_step', 5))   # TODO: Determine a reasonable value for this\n",
+    "epochs = int(os.environ.get('ds_epochs', 50))\n",
```

Comment:
Locally on my CPU I don't converge with `epochs=50`
```
STARTING Epoch 0049
Training model...
Training loss=57.992847443 avg_cer=0.326923072 WER: 0.909091
--------------------------------------------------------------------------------
    WER:    0.909091
    loss:   57.992847
    source: "she had your dark suit in greasy wash water all year"
    result: "sh hd rdark sut i grsa aw wae al year"
Hibernating training session into directory /Users/kdavis/.local/share/deepspeech
FINISHED Epoch 0049   Overall epoch time: 0:00:05   Training time: 0:00:03

Optimization Finished!
```
we should bumpy this to 75.